controllers/chat/ChatController.js

```diff
@@ -3,7 +3,7 @@ const Validator = require('fastest-validator');
 const {v4: uuidv4} = require('uuid');
 const {responseSuccess, responseError} = require('../../utils/Responses');
 
-const {User, ChatAnonUserInfo} = require('../../databases/models');
+const {User, ChatAnonUserInfo, Sequelize} = require('../../databases/models');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to include `Sequelize` from the `../../databases/models` module. This is necessary for using Sequelize's operators like `Sequelize.Op.ne` in the new code.
```diff
- const {User, ChatAnonUserInfo} = require('../../databases/models');
+ const {User, ChatAnonUserInfo, Sequelize} = require('../../databases/models');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->